opennms-bootstrap/src/main/java/org/opennms/bootstrap/Bootstrap.java

```diff
@@ -496,9 +498,9 @@ protected static void executeClass(final String classToExec, final String classT
         if (System.getProperty("org.opennms.protocols.icmp.interfaceJar") != null) {
             dir += File.pathSeparator + System.getProperty("org.opennms.protocols.icmp.interfaceJar");
         }
-        
-        if (System.getProperty("org.opennms.rrd.interfaceJar") != null) {
-            dir += File.pathSeparator + System.getProperty("org.opennms.rrd.interfaceJar");
+
+        if (System.getProperty("org.opennms.rrd.strategyClass", DEFAULT_RRD_STRATEGY_CLASS).equals("org.opennms.netmgt.rrd.rrdtool.MultithreadedJniRrdStrategy")) {
```

Comment:
Right side of `equals` could just be `DEFAULT_RRD_STRATEGY_CLASS`

Reply:
done.